development/acpica/acpica.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="acpica"
-VERSION="20170531"
+VERSION="20210331"
 HOMEPAGE="http://www.acpica.org"
-DOWNLOAD="https://acpica.org/sites/acpica/files/acpica-unix2-20170531.tar.gz"
-MD5SUM="46c4c93435ad1d8c4eb5bacde17503b8"
+DOWNLOAD="https://acpica.org/sites/acpica/files/acpica-unix2-20210331.tar.gz"
```

Comment:
I switched to the unix tarball, since the website notes the unix2 one is deprecated now

Reply:
Changed, thank you